docs/en/samples/index.md

```diff
@@ -43,43 +46,39 @@ A Postman collection is also available at the bottom of the page.
 ### Prerequisites
 
 * Download [Python 3](https://www.python.org)
-* 'Pip' is included as part of the download but it is recommended that you update it:
+* 'pip' is included as part of the download but it is recommended that you update it:
 ```
-pip3 install --upgrade pip
+pip install --upgrade pip
 ```
+* create a vitual environment (optional)
+```
+python -m venv path/to/ENV
+```
+activate it:
+```
+source path/to/ENV/bin/activate
+```
+* install dependencies
 
 All sample scripts use the Python packages `PyJWT`, `cryptography`, and `requests`. You must install these packages before running the scripts. You can install the packages with the following commands:
 ```
-pip3 install pyjwt
-pip3 install cryptography
-pip3 install requests
+pip install requests
 ```
+For DEPRECATED JWT workflows this modules need to be installed as well: _PyJWT_ and _cryptography_
 
 ### Setting Up the Environment
 
-Our sample directory includes a sample **usermanagement.config**. To make all the scripts more readable and adaptable, we use variables defined in this separate configuration file. To produce and send requests, the file defines the following values which you must update before attempting to run any of the samples:
+Each sample file contains at the top some variables that need to be initialised based on the information generated at Project creation phase, or specific to the action needed (like sample account info, or existing account's email):
 ```
-[server]
-host = usermanagement.adobe.io
-endpoint = /v2/usermanagement
-ims_host = ims-na1.adobelogin.com
-ims_endpoint_jwt = /ims/exchange/jwt
-
-[enterprise]
-domain = <my enterprise domain>
-fed_domain = <my federated domain>
-org_id = my organization id
-api_key = my api key/client id
-client_secret = my api client secret
-tech_acct = my api client technical account
-priv_key_filename = my private key filename
+ACCESS_TOKEN = ''
+CLIENT_ID = ''
+ORG_ID = ''
+# add below an existing account's email from Admin Console
+USER_EMAIL = ''
```

Comment:
I would recommend keeping with the same structure as before. It reduces changes for clients that already use these scripts.

Reply:
the samples as they were, are/were unusable in prod environments as they are. They lack call management logic or response processing, logging etc..
With that in mind, I do not think there is impact, since the only purpose is/was to demo the calls.